test/fixtures/integration/llhls-byte-range.js

```diff
@@ -0,0 +1,260 @@
+module.exports = {
+  allowCache: true,
+  discontinuitySequence: 0,
+  discontinuityStarts: [],
+  mediaSequence: 0,
+  playlistType: 'VOD',
+  preloadSegment: {
+    preloadHints: [
+      {
+        'BYTERANGE-LENGTH': 2000,
+        'TYPE': 'PART',
+        'URI': 'filePart273.3.mp4',
+        'byterange': {
+          length: 2000,
+          offset: 8118592
+        }
+      },
+      {
+        'BYTERANGE-LENGTH': 5000,
+        'BYTERANGE-START': 8355216,
+        'TYPE': 'PART',
```

Comment:
Is this valid due to https://tools.ietf.org/html/draft-pantos-hls-rfc8216bis-08#section-6.2.6 ?

> The Server SHOULD NOT add more than one EXT-X-PRELOAD-HINT tag with
the same TYPE to a Playlist.

(I suppose valid but not recommended, but may be worth identifying if so in the playlist name)